content/blog/2015-04-23-prometheus-posts-around-the-web.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Prometheus posts around the web"
+created_at: 2015-04-23
+kind: article
+author: Brian Brazil
+---
+
+It has been almost 3 months since Prometheus version [0.10.0 was publicly
+announced](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud).
+
+SoundCloud's post remains the best overview of the key components of
+Prometheus, but there has been lots of other online activity around Prometheus.
+This post will let you catch up on anything you missed.
+
+## Using Prometheus
+
+Posts on how to use Prometheus compose the majority of online content. Here's the
+ones we're aware of with part of the ecosystem they cover:
+
+* Container Exporter: [Monitor Docker Containers with Prometheus](https://5pi.de/2015/01/26/monitor-docker-containers-with-prometheus/)
+* HAProxy: [HAProxy Monitoring with Prometheus](http://www.boxever.com/haproxy-monitoring-with-prometheus)
+* Java Client: [Easy Java Instrumentation with Prometheus](http://www.boxever.com/easy-java-instrumentation-with-prometheus)
+* Java Client and Labels: [The Power of Multi-Dimensional Labels in Prometheus](http://www.boxever.com/the-power-of-multi-dimensional-labels-in-prometheus)
+* Node Exporter: [Monitoring your Machines with Prometheus](http://www.boxever.com/monitoring-your-machines-with-prometheus)
+* JMX Exporter: [Cassandra Consoles with JMX and Prometheus](http://www.boxever.com/cassandra-consoles-with-jmx-and-prometheus)
+* Python Client and Node Exporter Textfile Collector: [Monitoring Python Batch Jobs](http://www.boxever.com/monitoring-python-batch-jobs)
+* Mesos Exporter: [Monitoring Mesos tasks with Prometheus](http://www.antonlindstrom.com/2015/02/24/monitoring-mesos-tasks-with-prometheus.html)
+* Synapse: [Monitoring Synapse Metrics with Prometheus](http://matrix.org/blog/2015/04/23/monitoring-synapse-metrics-with-prometheus/)
+
+## Articles
+
+These articles look at how Prometheus fits into the broader picture of keeping services up and running:
+
+* [Prometheus: A Next-Generation Monitoring System](http://www.boxever.com/prometheus-a-next-generation-monitoring-system)
+* [SoundCloud’s Prometheus: A Monitoring System and Time Series Database Suited for Containers](http://thenewstack.io/soundclouds-prometheus-monitoring-system-time-series-database-suited-containers/)
+* [Docker Monitoring Continued: Prometheus and Sysdig](http://rancher.com/docker-monitoring-continued-prometheus-and-sysdig/)
+
+## Philosophy
+
+Monitoring isn't just about the technical details, how it affects the design of
+your systems, operations and human factors is important too:
```

Comment:
comma after "operations"

(maybe even after "factors" because of the way the sentence is constructed? - not sure)